jax/_src/scipy/linalg.py

```diff
@@ -103,7 +103,19 @@ def eigh(a, b=None, lower=True, eigvals_only=False, overwrite_a=False,
   del overwrite_a, overwrite_b, turbo, check_finite
   return _eigh(a, b, lower, eigvals_only, eigvals, type)
 
-
+@partial(jit, static_argnames=('output',))
+def _schur(a, output):
+  if output == "complex":
+    a = a.astype(jnp.complex64 if a.dtype <= jnp.complex64 else jnp.complex128)
+  return lax_linalg.schur(a)
+
+@_wraps(scipy.linalg.schur)
+def schur(a, output='real', lwork=None, overwrite_a=False, sort=None, check_finite=True):
+  del lwork, overwrite_a, sort, check_finite
```

Comment:
Can you omit unsupported arguments instead of deleting them? If someone explicitly writes `check_finite=True`, I would rather give them an `TypeError` about an invalid argument.

Reply:
Oh sorry misunderstood, I will fix it